packages/pds/src/account-manager/helpers/invite.ts

```diff
@@ -155,6 +155,30 @@ export const getAccountInviteCodes = async (
   }))
 }
 
+export const getAccountsInviteCodes = async (
```

Comment:
Can dry this up a bit & use this function to implement `getAccountInviteCodes`?

Reply:
decided to stick to the pattern of `take an array and return a map` and got rid of the singular `getAccountInviteCodes` in favor of `getAccountsInviteCodes`